docs/scripts.md

```diff
@@ -94,26 +102,20 @@ yarn build:app
 
 This command will build gatsby and the website.
 
-### Storybook
-
-```bash
-yarn build:storybook
-```
-
-This command will build our storybook instance.
-
 ### Lambda
 
 ```bash
 yarn build:lambda
 ```
 
-This command will build our meetup lambda.
+This command will build our lambdas using the [netlify-lambda](https://github.com/netlify/netlify-lambda) package.
+
+See [here](../README.md#ƛ-netlify-lambda-automated-deployments) for details on how the lambdas are used.
```

Comment:
Maybe write out the heading rather than the word 'here'? Just to more clearly indicate where you're sending the person.